gammapy/modeling/models/spectral.py

```diff
@@ -443,20 +446,31 @@ def plot(
         -------
         ax : `~matplotlib.axes.Axes`, optional
             Matplotlib axes.
+
+        Notes
+        -----
+        Either ``energy_bounds`` or ``energy_axis`` must be defined.
         """
         from gammapy.estimators.map.core import DEFAULT_UNIT
 
-        ax = plt.gca() if ax is None else ax
-
         if self.is_norm_spectral_model:
             sed_type = "norm"
 
-        energy_min, energy_max = energy_bounds
-        energy = MapAxis.from_energy_bounds(
-            energy_min,
-            energy_max,
-            n_points,
-        )
+        if energy_bounds is not None:
```

Comment:
If both are defined, which one to use? Why the priority is given to energy_bounds?